### PR TITLE
Explainer section on home page

### PR DESCRIPTION
**Pull request by jessieay**
* Includes section on home page that explains our service, mirroring
  GCF home page.

Reason for Change
===================
* Explain our service

Changes
=======
* Add new image and section to home page

screen shot 2017-08-07 at 11 16 59 am

### CODE REVIEW

**Comment by jessieay** [comment]
[Below the homepage hero image and primary CTA, include a section that explains our service, mirroring GCF homepage:](https://trello.com/c/qwZbpeE7/78-below-the-homepage-hero-image-and-primary-cta-include-a-section-that-explains-our-service-mirroring-gcf-homepage)

File: app/assets/stylesheets/templates/_homepage.scss [hunk 1]
```diff
@@ -1,4 +1,25 @@
 .template--homepage {
+  .trust-badge {
```
Comment (jessieay) [inline app/assets/stylesheets/templates/_homepage.scss]: NOTE: all CSS in this PR was taken directly from GCF (or adapted slightly as needed)

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -24,6 +25,7 @@
         </h4>
       </div>
       <div class="program-selector__subtitle text--smell">
+        MichiganBenefits.org is a non-profit offering online and in-person assistance to Michigan residents enrolling in the Food Assistance Program (food stamps). Most qualified households get more than $200 a month.
```
Comment (jayroh) [inline app/views/pages/index.html.erb]: Could we wrap this line? :D 

(pretty please)

File: app/views/pages/index.html.erb [hunk 3]
```diff
@@ -36,6 +38,28 @@
...
+  <div class="grid">
+    <div class="grid__item width-seven-twelfths">
+      <p>
+        MichiganBenefits.org is a non-profit offering online and in-person
+        assistance to Michigan residents enrolling in the Food Assistance Program
+        (food stamps). Most qualified households get more than $200 a month.
+      </p>
+
+      <p>
+      To visit the official online application for food assistance, please visit <%= link_to 'MI Bridges', 'https://www.michigan.gov/mibridges' %>.
```
Comment (jayroh) [inline app/views/pages/index.html.erb]: Consistent indentation

File: app/views/pages/index.html.erb (hunk 2 above)
Comment (jessieay) [inline app/views/pages/index.html.erb]: of course! forgot to, glad you saw this. done at 2b0c4af

File: app/views/pages/index.html.erb (hunk 3 above)
Comment (jessieay) [inline app/views/pages/index.html.erb]: also addressed at 2b0c4af :)

**Pull request merged**